### PR TITLE
Gitignore and data link

### PR DESCRIPTION
**Pull request by egrace479**
- Removed `.DS_Store` files in repo and added a `.gitignore` for extraneous Python and Mac system files. 
- Added link in README to GS Hoyal Cuthill dataset on HF.

@reshmarameshbabu please let us know if you have any concerns.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ The big biological question of connecting phenotype to genotype shapes the futur
 - Custom 15 layer CNN + triplet loss
 - Refer to metadata for files used within code
 - 24 subspecies, 12 mimic pairs
+- Full data is available on the [Imageomics Hugging Face](https://huggingface.co/datasets?other=doi%3A10.57967%2Fhf%2F1351).
```
Comment (hlapp) [inline README.md]: Probably better to use the DOI link if it's effectively linking by DOI anyway:
```suggestion
- Full data is available on the [Imageomics Hugging Face](https://doi.org/10.57967/hf/1351).
```

**Review by hlapp — DISMISSED** [review]
See one inline change. Other than that it's all 👍

**Review by hlapp — APPROVED** [review]

**Review by thompsonmj — APPROVED** [review]
LGTM

**Comment by egrace479** [comment]
@reshmarameshbabu please let us know if you have any concerns, otherwise I will be merging this PR tomorrow (Tues, Nov. 28) afternoon.

**Pull request merged**